test/native/helpers.js

```diff
@@ -97,7 +75,42 @@ export async function initializeEditor( props, { component = Editor } = {} ) {
 	global.requestAnimationFrame = originalRAF;
 	global.cancelAnimationFrame = originalCAF;
 
-	return screen;
+	return result;
+}
+
+/**
+ * Initialize an editor for test assertions.
+ *
+ * @param {Object}                    props               Properties passed to the editor component.
+ * @param {string}                    props.initialHtml   String of block editor HTML to parse and render.
+ * @param {Object}                    [options]           Configuration options for the editor.
+ * @param {import('react').ReactNode} [options.component] A specific editor component to render.
+ * @return {import('@testing-library/react-native').RenderAPI} A Testing Library screen.
+ */
+export async function initializeEditor( props, { component = Editor } = {} ) {
+	return waitForStoreResolvers( () => {
```

Comment:
The usages of `waitForStoreResolvers` look good to me. Thanks for taking it into consideration. I appreciate it.